classloader-leak-prevention/classloader-leak-prevention-core/pom.xml

```diff
@@ -87,7 +87,7 @@
     <dependency>
       <groupId>org.geotools</groupId>
       <artifactId>gt-metadata</artifactId>
-      <version>2.6.2</version>
```

Comment:
Is this actualy working for you, @davoustp ?

Reply:
Yes it is.
But the `gt-metadata` artifact has moved to another repo, and not available in public Maven: http://maven.geo-solutions.it/
I struggled quite a bit to find this one, to be honest. :-)